azure-kusto-data/tests/e2e.py

```diff
@@ -300,3 +303,33 @@ def test_cloud_info(self):
     def test_cloud_info_404(self):
         cloud_info = CloudSettings.get_cloud_info_for_cluster("https://www.microsoft.com")
         assert cloud_info is CloudSettings.DEFAULT_CLOUD
+
+    def test_redirects(self):
+        redirect_codes = [301, 302, 303, 307, 308]
+        with KustoClient("https://httpstat.us/") as client:
+            for code in redirect_codes:
+                client._query_endpoint = f"https://httpstat.us/{code}"
+                with pytest.raises(TooManyRedirects):
+                    client.execute("db", "table")
+            client.set_redirect_count(5)
```

Comment:
nit: a new line after scope back would be nice :)

Reply:
Fixed